planner/core/task.go

```diff
@@ -649,6 +649,9 @@ func finishCopTask(ctx sessionctx.Context, task task) task {
 			indexPlan:      t.indexPlan,
 			ExtraHandleCol: t.extraHandleCol,
 		}.Init(ctx, t.tablePlan.SelectBlockOffset())
+		if tp, ok := p.tablePlan.(*PhysicalTableScan); ok {
```

Comment:
What if the `tablePlan` is `Selection->TableScan`?

Reply:
I didn't take this scenario into account, will fix it.